cmd/web.go

```diff
@@ -40,9 +40,14 @@ and it takes care of all the other things for you`,
 			Value: "3000",
 			Usage: "Temporary port number to prevent conflict",
 		},
+		cli.StringFlag{
```

Comment:
Are you able to add `--custom-path` to the other cli cmds, and add the changes made to `--config` below to the others as well.

Also could you update the https://docs.gitea.io/en-us/command-line/ docs page?

Reply:
OK I've done the opposite. I've removed --config from all of the subcommands.

But I've also made all global options work on the subcommands - so I've also done what you asked.

Command-line docs page has also been updated.